tests/integration/tls_tests/test_tls.py

```diff
@@ -84,6 +147,8 @@ async def test_rotate_tls_key(ops_test: OpsTest) -> None:
         new_internal_cert_time = await time_file_created(ops_test, unit.name, INTERNAL_CERT_PATH)
         new_mongod_service_time = await time_process_started(ops_test, unit.name, DB_SERVICE)
 
+        check_certs_correctly_distributed(ops_test, unit)
```

Comment:
Hmm, I'm also thinking to add this to the following test, `test_set_tls_key()`, where we're setting specific keys.